Hashtable.c

```diff
@@ -120,11 +219,17 @@ void* Hashtable_remove(Hashtable* this, unsigned int key) {
          }
       }
    }
+
    assert(Hashtable_isConsistent(this));
+
+   // shrink on load < 0.2
```

Comment:
`load` is a well defined term for a tool that is used in systems monitoring, better use `fill level` 2/2